src/client/src/utils/toSellSideView.ts

```diff
@@ -0,0 +1,29 @@
+import { ROUTES_CONFIG } from "@/constants"
+import { constructUrl } from "./url"
+import { openWindow } from "./window/openWindow"
+
+export const RFQ_ID_PARAM = "rfqId"
+
+let windowReference: Window | undefined
+
+export const toSellSideView = async (rfqId: number | null) => {
```

Comment:
I'd prefer something like `showRfqInSellSide` .. this looks like it is going to be a (data) transformer function, from outside .. to me, anyway

Reply:
neat mechanism though 👍🏻